packages/core/i18n/core-i18n-browser-mocks/src/i18n_context_mock.test.tsx

```diff
@@ -21,14 +21,4 @@ describe('I18nProviderMock', () => {
       ).html()
     ).toMatchInlineSnapshot(`"default message"`);
   });
-
-  it('interpolates to id if default message is not present', () => {
-    expect(
-      shallow(
-        <I18nProviderMock>
-          <FormattedMessage id="id" />
-        </I18nProviderMock>
-      ).html()
-    ).toMatchInlineSnapshot(`"id"`);
-  });
```

Comment:
What is the behavior now then 😅 (or is it just that `defaultMessage` is no longer optional on `<FormattedMessage/>`?

Reply:
Correct it was never optional, the i18n_check fails if `id` or `defaultMessage` is missing